### PR TITLE
ci: updates elixir/otp tested versions

### PR DESCRIPTION
**Pull request by grzuy**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,11 +11,11 @@ jobs:
...
         include:
-          - elixir_version: 1.15.4
-            otp_version: 26.0.2
+          - elixir_version: 1.16.1
+            otp_version: 26.2.2
             lint: true
-          - elixir_version: 1.14.5
-            otp_version: 25.3.2.5
+          - elixir_version: 1.15.7
+            otp_version: 25.3.2.9
```
Comment (jonatanklosko) [inline .github/workflows/ci.yml]: The idea is to run on the latest and oldest supported version :)

```suggestion
          - elixir_version: 1.14.5
            otp_version: 25.3.2.5
```

**Review by jonatanklosko — APPROVED** [review]
Thanks, I missed a notification for this PR apparently :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (grzuy) [reply .github/workflows/ci.yml]: right, 1.14 still listed in the `mix.exs`, good call

**Comment by grzuy** [comment]
Thank you!

**Pull request merged**